dev-tools/mess-detector/phpmd.baseline.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0"?>
+<phpmd-baseline>
+  <violation rule="PHPMD\Rule\Controversial\CamelCaseClassName" file="tests/Fixtures/ConfigurationResolverPathsIntersection/d/.php-cs-fixer.php"/>
+  <violation rule="PHPMD\Rule\Controversial\CamelCaseClassName" file="tests/Fixtures/FixerTest/fix/somefile.php"/>
+  <violation rule="PHPMD\Rule\Controversial\CamelCasePropertyName" file="tests/Fixtures/FixerTest/fix/somefile.php"/>
```

Comment:
imho let's ignore all fixtures
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7224/files#r1291342637

Reply:
no opinion on that